docker-compose.yml

```diff
@@ -27,17 +27,17 @@ x-container-common: &container-common
     - quorum
     - besu
     - hashicorp
-    - qkm
+    - postgres
```

Comment:
postgres is a mandatory dependency of the qkm, so qkm is correct here.

Reply:
ok